README.md

```diff
@@ -208,6 +208,18 @@ source activate nums
 pip install -e ".[testing]"
 ```
 
+For Windows, follow the steps below if you do not have Anaconda already downloaded. 
+1. Follow the steps to download the Anaconda Installer. https://docs.anaconda.com/anaconda/install/windows/
+2. During the setup, select that you want Anaconda to be added to the PATH variable. 
+3. Fork/Clone the nums-project repository
+4. Open Command Prompt and follow the instructions below:
+    ```sh
```

Comment:
Can you remove this sh block and just update the one above from `source activate nums` to `conda activate nums`?

Reply:
So, do you still want two different blocks of instructions - one for Linux/Mac and one for Windows, as per comment above.

Reply:
Yes, create a separate section in "Installation" for Windows, and let's see what it looks like. If it's just a couple of sentences, then we can simplify.

In this section ("Contributing") it doesn't look like we need a separate Windows-based set of instructions.

Alternatively, I can help with this part of the PR based on what you've written thus far if you give me push permissions to your repo.

Reply:
Okay sure. I have given you access, I think you have a better idea of how the final set of instructions should work.